gcc/config.gcc

```diff
@@ -1190,7 +1190,6 @@ case ${target} in
 aarch64*-*-elf | aarch64*-*-fuchsia* | aarch64*-*-rtems*)
 	tm_file="${tm_file} elfos.h newlib-stdint.h"
 	tm_file="${tm_file} aarch64/aarch64-elf.h aarch64/aarch64-errata.h aarch64/aarch64-elf-raw.h"
-	tmake_file="${tmake_file} aarch64/t-aarch64"
```

Comment:
This was removed to be added for all targets at once. Need to double check whether this still happens or it is a regrssion...